Content munge devportal (#11982)
hashicorp/packer: website/content/docs/datasources/index.mdx

# Data Sources

Data sources let Packer fetch data to use in a template, including information defined outside of Packer.

> Refer to the [`data`](/docs/templates/hcl_templates/datasources) block documentation to learn more about working with data sources. The documentation also contains a page for each type of data source.

Author: ```suggestion
Refer to the [`data`](/docs/templates/hcl_templates/datasources) block documentation to learn more about working with data sources. The documentation also contains details about each type of data source.
```
